### ISSUE TITLE
Clear button on Select is not accessibile

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#5939]
### Expected Behavior
The user should be able to navigate to the clear button on Select using the keyboard.

### Actual Behavior
You cannot reach the clear button using the keyboard.

### URL, screen shot, or Codepen exhibiting the issue
See Select/Clear story https://storybook.grommet.io/?path=/story/input-select-clear--clear

### Steps to Reproduce
Try navigating with the keyboard to the clear button on the above story

### Your Environment
- Grommet version: 2.20.1
